### ISSUE TITLE
Zarmazd lists incorrect skill requirements

### ISSUE THREAD

**Issue by TomRichter** [peterhaneve/evemon#166]
Using EVEMon v4.0.11

In the skill planner, the Zarmazd incorrectly requires Gallente Cruiser 5 instead of Precursor Cruiser 5.  By a quick skim, no other Precursor ship seems to be affected.

![image](https://user-images.githubusercontent.com/11639332/56483965-2251b780-649b-11e9-89de-28fc1fd05a0c.png)

**Comment by peterhaneve** [comment]
Attributes like this one are generated using an automated process from CCP provided data. It looks like I used a bugged version of the CCP SDE to build the Zarmazd, which appears to have since been fixed. While the change is easy to make, I am really waiting for the SDE to be updated (it has been two months and several major patches since they last updated it) to release 4.12.

Unfortunately it may be the case that CCP has stopped providing this file. Re-engineering EVEMon to pull the data from ESI upon release would be difficult, not to mention the ESI server strain from the tens of thousands of requests this would cause.
